tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
-# installation
+# Installation
+
+This tutorial describes how to install a Marble node on a server. 
+
+## Prerequisites
+
+### Docker
+
+Marble is deployed as a [Docker](https://www.docker.com/) application using the 
+[compose](https://docs.docker.com/compose/) tool to manage and various services that make us the application.
+
+To install Docker on Unix/Linux either install [Docker Engine](https://docs.docker.com/engine/install/) (recommended) or
+[Docker Desktop](https://docs.docker.com/desktop/). If using an operating system other than Unix/Linux, you must use
+Docker Desktop.
+
+```{warning}
+We highly recommend installing Marble on a Unix/Linux machine. All code is developed and tested on Unix operating
+systems, and we cannot guarantee that the software will behave as expected on other operating systems.
+```
```

Comment:
PAVICS claims that it works on: 

Centos 7, RockyLinux 8, Ubuntu 18.04, 20.04, 22.04

(https://birdhouse-deploy.readthedocs.io/en/latest/deployment.html#id5)

But I guarantee this information is out of date. The idea with deploying it with docker is that it should (theoretically) all work as intended with an up to date version of Docker Engine (for Linux).

We don't have the resources to test on multiple different OS's at the moment. Let me reword this a bit to make the claim a bit more vague to cover our butts.

Reply:
This looks good. I agree about the Docker part, my thinking was more related to the shell scripts that come included with birdhouse-deploy.

Reply:
Yeah I agree. The scripts are supposed to be written as POSIX compliant as possible. If there's anything that is bash specific for example, they run it in a docker container that runs bash. Theoretically, all scripts should run properly in all shells that implement a superset of POSIX. But again, since we can't test thoroughly, you're right that we shouldn't make that claim.

Reply:
From my experience, I recall that I had encountered a minor issue when trying to deploy it on a Mac, and I think it was related to a shell script function.

Reply:
Was it an issue with the `readlink` function? The mac version is different and it always trips me up. It's really annoying

Reply:
Don't recall specifically... it was back in spring last year.